### ISSUE TITLE
Issue with piping from pysradb search to pysradb download

### ISSUE THREAD

**Issue by bscrow** [saketkc/pysradb#44]
* pysradb version: pysradb 0.10.5-dev0
* Python version: Python 3.7
* Operating System: Windows 10 Pro 1909

### Description

When I was trying to pipe the output from pysradb search to pysradb download, a FileNotFoundError was raised.

### What I Did

```
pysradb search -m 100 -q ribosome profiling --db ena -v 1 | pysradb download
```
![image](https://user-images.githubusercontent.com/38070760/85671104-22c78d00-b6f4-11ea-84b9-9c8f2c99050b.png)

**Comment by saketkc** [comment]
I just pushed https://github.com/bscrow/pysradb/commit/cffdd78d03aba8fd355f57301a5e3bf4cc498d8e to your fork. It is meant as a fix, but doesn't work yet. I was not able to get the downloads to work on ftp mode. Would you be willing to give it a shot?

**Comment by saketkc** [comment]
Any updates on this @bscrow?

**Comment by bscrow** [comment]
Fixed in #57
